test/qtismtest/data/storage/xml/XmlUtilsTest.php

```diff
@@ -391,6 +391,52 @@ public function testFindCustomNamespaces(): void
         );
     }
 
+    public function testValueAsStringReplaceSpecialSymbols(): void
+    {
+        $xml = ('<value>160°</value>');
+        $this->assertEquals('&lt;value&gt;160°�&lt;/value&gt;', Utils::valueAsString($xml));
```

Comment:
```suggestion
        $xml = ("160\b");
        $this->assertEquals("160\u{FFFD}"", Utils::valueAsString($xml));
```

This will make it a lot easier to understand the intent IMO.